[WIP] UI for Ansible Credentials

mzazrivec opened the pull request.

h-kataria commented:
@mzazrivec do we already have product features for this, i don't see them in https://github.com/ManageIQ/manageiq/pull/14020

h-kataria commented:
@mzazrivec just curious if value for Type column on list of Credentials is coming from the db, looked weird to have "Credentials (xxx)" in the list instead of just the type.

mzazrivec commented:
@h-kataria We already have `embedded_automation_manager_credentials*` features
added previously.

The credential types come from the model type, as I added them into `en.yml` in
https://github.com/ManageIQ/manageiq/pull/14020

martinpovolny commented:
As I wrote on gitter. I don't like any of the changes to `app/helpers/application_helper.rb`. But fixing that is not a topic of this PR. It just shows that we still have stuff to fix before we can claim that features can be added to the UI w/o dealing with spaghetti.
